common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -53,7 +53,9 @@ public enum Feature {
 
         ACCOUNT_V3("Account Console version 3", Type.DEFAULT, 3, Feature.ACCOUNT_API),
 
-        ADMIN_FINE_GRAINED_AUTHZ("Fine-Grained Admin Permissions", Type.PREVIEW),
+        ADMIN_FINE_GRAINED_AUTHZ("Fine-Grained Admin Permissions", Type.PREVIEW, 1),
```

Comment:
We should have this feature starting as `EXPERIMENTAL`.

Reply:
The `v1` version should be the default when starting the server using `fine-grained-admin, doesn't it?

Reply:
> The `v1` version should be the default when starting the server using `fine-grained-admin, doesn't it?

The preview feature should take a precedence over experimental, if unversioned key is used: https://github.com/keycloak/keycloak/blob/ea5e623b367a27511d881249695077d860559f4c/common/src/main/java/org/keycloak/common/Profile.java#L325-L336

But it seems the current behavior is not correct, I'll try to include the fix as part of this PR

Reply:
with the second commit, it should be fixed